aip/general/0162.md

```diff
@@ -27,199 +27,174 @@ to the version of an API as a whole.
+`latest`), which are read-only and managed by the service.
+
+- If a `latest` tag exists, it **must** represents the most recent created
+  revision.
+ The content of `publishers/{publisher}/books/{book}/revisions/latest` and
+  `publishers/{publisher}/books/{book}` can differ. Because the latest revision
+  may be different from the current state of the resource.
+
+```
+GET /v1/publishers/{publisher}/books/{book}/revisions/{reserved_keyword}
+```
+
+#### User specified
+
+User specified tags are tags provided by the user when tagging specific resource
+revision.
+
 APIs implementing resource revisions **may** provide a mechanism for users to
 tag a specific revision with a user provided name by implementing a "Tag
 Revision" custom method:
 
 ```proto
 rpc TagBookRevision(TagBookRevisionRequest) returns (Book) {
   option (google.api.http) = {
-    post: "/v1/{name=publishers/*/books/*}:tagRevision"
+    post: "/v1/{name=publishers/*/books/*/revisions/*}:tag"
     body: "*"
   };
 }
 ```
 
 ```proto
 message TagBookRevisionRequest {
-  // The name of the book to be tagged, including the revision ID.
+  // The name of the book revision to be tagged.
   string name = 1 [
     (google.api.field_behavior) = REQUIRED,
     (google.api.resource_reference) = {
-      type: "library.googleapis.com/Book"
+      type: "library.googleapis.com/BookRevision"
     }];
 
-  // The tag to apply.
-  // The tag should be at most 40 characters, and match `[a-z][a-z0-9-]{3,38}[a-z0-9]`.
+  // The tag to apply. The tag should be at most 40 characters, and match
+  // `[a-z][a-z0-9-]{3,38}[a-z0-9]`.
   string tag = 2 [(google.api.field_behavior) = REQUIRED];
 }
 ```
-
-- The `name` field **should** require an explicit revision ID to be provided.
+- The `name` field **should** require an explicit resource revision to be
+  provided.
   - The field **should** be [annotated as required][aip-203].
   - The field **should** identify the [resource type][aip-123] that it
     references.
 - The `tag` field **should** be [annotated as required][aip-203].
   - Additionally, tags **should** restrict letters to lower-case.
 - Once a revision is tagged, the API **must** support using the tag in place of
   the revision ID in `name` fields.
-  - If the user sends a tag, the API **must** return the tag in the resource's
-    `name` field, but the revision ID in the resource's `revision_id` field.
-  - If the user sends a revision ID, the API **must** return the revision ID in
-    both the `name` field and the `revision_id` field.
+  - If the user sends a tag or revision ID, the API **must** return the revision
+    ID in the resource's `name` field.
 - If the user calls the `Tag` method with an existing tag, the request **must**
   succeed and the tag updated to point to the new requested revision ID. This
   allows users to write code against specific tags (e.g. `published`) and the
   revision can change in the background with no code change.
-
 ### Listing revisions
```

Comment:
maybe have this up by "Getting revisions"? It's just enumerating resource-oriented requirements.